src/test/tpc/tpch_test.cpp

```diff
@@ -44,11 +44,16 @@ class TPCHTest : public BaseTestWithParam<TPCHTestParam> {
       {1, 0.01f},   {2, 0.004f},  {3, 0.01f},  {4, 0.005f},  {5, 0.01f},    {6, 0.01f},  {7, 0.01f},  {8, 0.01f},
       {9, 0.01f},   {10, 0.02f},  {11, 0.01f}, {12, 0.01f},  {13, 0.01f},   {14, 0.01f}, {15, 0.01f}, {16, 0.01f},
       {17, 0.013f}, {18, 0.005f}, {19, 0.01f}, {20, 0.008f}, {21, 0.0075f}, {22, 0.01f}};
+
+  // Helper method used to touch TPCHBenchmarkItemRunner's privates
```

Comment:
I would... [rephrase that comment](https://www.dict.cc/englisch-deutsch/privates.html)

Reply:
Oh. Oups. Yes, I would never make such an immature joke. ^^